packages/components/src/test/properties/UniquePropertyValuesTargetSelector.test.tsx

```diff
@@ -168,17 +146,36 @@ describe("UniquePropertyValuesTargetSelector", () => {
     const value = convertToPropertyValue(displayValue, groupedRawValues);
 
     const { queryByText } = render(
-      <UniquePropertyValuesTargetSelector
+      <UniquePropertyValuesSelector property={propertyDescription} onChange={() => {}} imodel={testImodel} descriptor={descriptor} value={value} />,
+    );
+
+    expect(queryByText(displayValue[0])).to.not.be.null;
+    expect(queryByText(displayValue[1])).to.not.be.null;
+  });
+
+  it("does not set value when provided value is not parsable", () => {
```

Comment:
```suggestion
  it("does not set value when provided value is invalid", () => {
```

Reply:
for some reason can't commit suggestion, fixed